model_training/local_prediction/README.md

```diff
@@ -52,3 +51,13 @@ Tested on
    ```bash
    mean average precision (mAP@0.50) = 0.823387, or 82.34 %
    ```
+
+## Training Locally
+
+1. Make sure requirements are setup as on the [darknet repo](https://github.com/AlexeyAB/darknet#requirements-for-windows-linux-and-macos). NVIDIA GPU is required.
+1. Follow the steps in `../Train_yolov4_tiny.ipynb` to configure envinronment variables, hyperparameters, and run scripts.
```

Comment:
Can probably add a link to the ipynb too

Reply:
Also, the file paths aren't consistent in this README. Some of them use `\` while others use `/`. I would suggest standardizing to `/`